axiomapy/axiomaapi/clienteventbus.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright © 2022 Qontigo GmbH.
```

Comment:
do you want to check whether to change the "Qontigo"?

Reply:
I can bring this up in next axioma-py meeting!